enterprise_access/apps/api_client/tests/test_utils.py

```diff
@@ -15,3 +21,106 @@ def __init__(self, json_data, status_code):
 
     def json(self):  # pylint: disable=arguments-differ
         return self.json_data
+
+
+class TestLicenseManagerMetadata(TestCase):
```

Comment:
nit: this class isn't testing anything per se, so perhaps the name should be something more like `MockLicenseManagerMetadataMixin`?